drivers/sensor/sensor.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <dm.h>
+#include <error.h>
+#include <sensor.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+int
+sensor_get_value(struct device *dev, uint8_t id, uint16_t *value)
```

Comment:
`value` needs to be a `uint32_t` (or we're limited to 65.535°C)

Reply:
done